Makefile

```diff
@@ -214,6 +218,14 @@ olm-deploy: operator-sdk ## Deploy operator via OLM
 olm-upgrade: operator-sdk ## Upgrade operator via OLM
 	$(OPERATOR_SDK) run bundle-upgrade -n $(OPERATOR_NAMESPACE) $(BUNDLE_IMG)
 
+##@ OpenTelemetry Operator deployment
+.PHONY: otel-deploy
+otel-deploy: operator-sdk ## Deploy OpenTelemetry operator via OLM
+	kubectl create namespace otel-operator-system
+        # Wait for OLM components to be ready
```

Comment:
Why does the `otel-deploy` target need to wait for the OLM installation, shouldn't the place where OLM is installed wait for completion?